leapp/repository/actor_definition.py

```diff
@@ -242,19 +244,10 @@ def injected_context(self):
         if self.tools:
             os.environ['LEAPP_TOOLS'] = os.path.join(self._repo_dir, self._directory, self.tools[0])
 
-        # We make a snapshot of the symbols in the module
```

Comment:
For anyone reviewing - The library snapshotting code was actually never really useful and didn't help - The new approach is much more resilient due to python handling most of the problematic cases :-)